src/renderer/layouts/default-layout/player-bar.tsx

```diff
@@ -1,15 +1,38 @@
+import { useState } from 'react';
 import styled from 'styled-components';
 import { Playerbar } from '/@/renderer/features/player';
+import { useGeneralSettings } from '/@/renderer/store/settings.store';
 
-const PlayerbarContainer = styled.footer`
+interface PlayerbarContainerProps {
+    barState: string;
```

Comment:
To accomodate the hover style, you can send a boolean value rather than the css variable string, and then conditionally choose either `playerbar-bg-active` or `playerbar-bg` depending on the boolean value.

This also needs to be a styled-components `transient` prop which uses the dollar sign `$`. If you don't add the `$`, the prop will be passed on as a HTML DOM element which for custom prop names would be invalid.

[Reference on the docs](https://styled-components.com/docs/api#using-custom-props)